src/ui/Forms/GetDictionaries.cs

```diff
@@ -12,9 +13,33 @@ namespace Nikse.SubtitleEdit.Forms
 {
     public sealed partial class GetDictionaries : Form
     {
-        private List<string> _dictionaryDownloadLinks = new List<string>();
-        private List<string> _descriptions = new List<string>();
-        private List<string> _englishNames = new List<string>();
+        private class DictionaryItem
+        {
+            /// <summary>
+            /// English name
+            /// </summary>
+            public string Name { get; set; }
+            /// <summary>
+            /// Native Name
+            /// </summary>
+            public string NativeName { get; set; }
+            public string Description { get; set; }
+            public string DownloadLink { get; set; }
+
+            public override string ToString()
+            {
+                var displayName = $"{Name}{(string.IsNullOrEmpty(NativeName) ? "" : $" - {NativeName}")}";
+                if (displayName.Length > 55)
```

Comment:
Could you use `comboBoxDictionaries.DropDownWidth` instead of 55?

Reply:
`comboBoxDictionaries.Width or comboBoxDictionaries.DropDownWidth` is not working properly.

I also tried to use `TextWiddth.CalcPixelWidth` but there are way too different in value when compared with `comboBoxDictionaries.DropDownWidth`